root/etc/e-smith/events/actions/nethserver-ntopng-3-migrate

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2020 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+# Migrate data from ntopng 3, if any
+
+if [ -d /var/tmp/ntopng ]; then
+  systemctl stop ntopng
+  rm -rf /var/lib/ntopng/
+  mv /var/tmp/ntopng /var/lib/ntopng/
+  chown -R ntopng:ntopng /var/lib/ntopng/
+  systemctl start ntopng
```

Comment:
This line will start ntopng even if it's disabled.
Before starting, check if is enabled.